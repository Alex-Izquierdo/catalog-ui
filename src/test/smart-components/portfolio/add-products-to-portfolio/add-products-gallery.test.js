import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import AddProductsGallery from '../../../../smart-components/portfolio/add-products-to-portfolio/add-products-gallery';

describe('<AddProductsGallery />', () => {
  let initialProps;
  beforeEach(() => {
    initialProps = {};
  });

  it('should render correctly', () => {
    const wrapper = shallow(<AddProductsGallery {...initialProps} />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
