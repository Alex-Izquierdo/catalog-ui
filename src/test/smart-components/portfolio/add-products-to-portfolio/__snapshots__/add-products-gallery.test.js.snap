// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddProductsGallery /> should render correctly 1`] = `
<ContentGallery
  checkedItems={Array []}
  editMode={true}
  renderEmptyState={[Function]}
/>
`;
